WordPress/src/main/java/org/wordpress/android/util/config/RemoteConfig.kt

```diff
@@ -12,20 +12,38 @@ import javax.inject.Inject
  * Do not use this class outside of this package. Use [AppConfig] instead
  */
 class RemoteConfig
-@Inject constructor() {
-    fun refresh() {
+@Inject constructor(private val appPrefsWrapper: AppPrefsWrapper) {
+    fun init() {
         val firebaseRemoteConfig = FirebaseRemoteConfig.getInstance()
         val configSettings = Builder()
                 .setMinimumFetchIntervalInSeconds(BuildConfig.REMOTE_CONFIG_FETCH_INTERVAL)
                 .build()
         firebaseRemoteConfig.setConfigSettingsAsync(configSettings)
         firebaseRemoteConfig.setDefaultsAsync(RemoteConfigDefaults.remoteConfigDefaults)
-        firebaseRemoteConfig.fetchAndActivate()
-                .addOnCompleteListener { task: Task<Boolean?> ->
+        firebaseRemoteConfig.activate()
```

Comment:
Maybe we could attach an `addOnCompleteListener` to this as well? 

Besides that I think it would also be interesting to track something similar to the `feature_flag_set` event, but only for activated Firebase values. It seems that the Task result [returns `true` if the current call activated the fetched configs](https://firebase.google.com/docs/reference/android/com/google/firebase/remoteconfig/FirebaseRemoteConfig?authuser=1#public-taskboolean-activate) so we could check if that's `true` and then track something like a `firebase_configs_activated` event with all the current activated configs, which we could retrieve from [`FirebaseRemoteConfig::getAll`](https://firebase.google.com/docs/reference/android/com/google/firebase/remoteconfig/FirebaseRemoteConfig?authuser=1#public-mapstring,-firebaseremoteconfigvalue-getall). We could then do the same for the listener attached to `fetchAndActivate`. 

I'm thinking that tracking something like this would allow us to accurately check what is coming from Firebase (before we introduce our own logic) and, unless I'm missing something, this event would necessarily have to show a 50/50 split on Tracks, so we could check that as well. Let me know what you think!

Reply:
I've added the `addOnCompleteListener` in a752e10. I've added tracking of the current sync state in 85fb47a
I think it makes sense 👍. I'm wondering if `feature_flag_set` still makes sense. Since we no longer cache the values, I've changed it to `feature_flag_value` but it's now tracked every time we read the value. I can keep it for now but maybe we should reconsider if it's not unnecessary noise.